src/wasm-binary.h

```diff
@@ -712,7 +712,7 @@ class WasmBinaryBuilder {
 
   // gets a name in the combined function import+defined function space
   Name getFunctionIndexName(Index i);
-  void getResizableLimits(Address& initial, Address& max, Address defaultIfNoMax);
+  void getResizableLimits(Address& initial, Address& max, bool& shared, Address defaultIfNoMax);
```

Comment:
why not put it at the end? seems more natural there at first glance, maybe there's something I'm missing?